src/vectorpostprocessors/ResponseHistoryMean.C

```diff
@@ -0,0 +1,81 @@
+// MOOSE includes
+#include "ResponseHistoryMean.h"
+#include "PostprocessorInterface.h"
+#include "VectorPostprocessorInterface.h"
+#include "MastodonUtils.h"
+#include "ResponseHistoryBuilder.h"
+/* #include <iostream>
+#include <fstream>
+using namespace std;*/
+
+registerMooseObject("MastodonApp", ResponseHistoryMean);
+
+template <>
+InputParameters
+validParams<ResponseHistoryMean>()
+{
+  InputParameters params = validParams<GeneralVectorPostprocessor>();
+  params.addRequiredParam<VectorPostprocessorName>(
+      "vectorpostprocessor",
```

Comment:
I suggest changing the name of this to "response_history" to make it clear what is expected.

Reply:
We discussed Friday on sticking to params since I was getting some errors.